docs/content/configuration/csi_ug_config_advanced_importvol.md

```diff
@@ -37,128 +37,121 @@ To find the `volumeHandle`, use one of the following procedures:
 
-1.  Create a persistent volume (PV) yaml file.
+1. Create a persistent volume (PV) YAML file.
 
     **Important:** Be sure to include the `storageClassName` and `controllerPublishSecretRef` parameters or errors will occur.
 
-2.  Take the `volume_name` and other optional information (collected before the procedure) and insert it into the yaml file.
-
-    <pre>
-    apiVersion: v1
-    kind: PersistentVolume
-    metadata:
-      # annotations:
-        # pv.kubernetes.io/provisioned-by: block.csi.ibm.com
-      name: vol1-pv
-    spec:
-      accessModes:
-      - ReadWriteOnce
-      capacity:
-        storage: 1Gi
-      csi:
-        controllerPublishSecretRef:
-          name: demo-secret
-          namespace: default
-        driver: block.csi.ibm.com
-        <b># volumeAttributes:
-          # pool_name: ibmc-block-gold
-          # storage_type: SVC
-          # volume_name: vol1
-          # array_address: baremetal10-cluster.xiv.ibm.com
-        volumeHandle: SVC:600507640082000B08000000000004FF</b>
-      # persistentVolumeReclaimPolicy: Retain
-      storageClassName: ibmc-block-gold
-      # volumeMode: Filesystem
-    </pre>
-
-3.  Create a PersistentVolumeClaim (PVC) yaml file.
+2. Take the `volume_name` and other optional information (collected before the procedure) and insert it into the YAML file (under `spec.csi.volumeAttributes`).
+
+    **Important:** If using the CSI Topology feature, the `spec.csi.volumeHandle` contains the system ID. In the example below, the `spec.csi.volumeHandle` would read similar to the following: `SVC:demo-system-id-1:600507640082000B08000000000004FF`.
+    
+        apiVersion: v1
+        kind: PersistentVolume
+        metadata:
+          # annotations:
+            # pv.kubernetes.io/provisioned-by: block.csi.ibm.com
+          name: vol1-pv
+        spec:
+          accessModes:
+          - ReadWriteOnce
+          capacity:
+            storage: 1Gi
+          csi:
+            controllerPublishSecretRef:
+              name: demo-secret
+              namespace: default
+            driver: block.csi.ibm.com
+            # volumeAttributes:
+              # pool_name: ibmc-block-gold
+              # storage_type: SVC
+              # volume_name: vol1
+              # array_address: baremetal10-cluster.xiv.ibm.com
+            volumeHandle: SVC:600507640082000B08000000000004FF
+          # persistentVolumeReclaimPolicy: Retain
+          storageClassName: ibmc-block-gold
+          # volumeMode: Filesystem
+
+3. Create a PersistentVolumeClaim (PVC) YAML file.
 
     **Note:**
 
-    -   To include a specific 5 Gi PV, be sure to include the `storageClassName`.
-    -   For more information about creating a PVC yaml file, see [Creating a PersistentVolumeClaim (PVC)](csi_ug_config_create_pvc.md).
+    - To include a specific 5 Gi PV, be sure to include the `storageClassName`.
```

Comment:
how does the `storageClassName` related to `a specific 5 Gi PV`? and where did the `5` come from?

Reply:
@oriyarde  This was originally requested by Matt (CSI-473).
Note that this was also reviewed and tested for last release by @tyichye19  (CSI-3082)

Reply:
thanks @rivkap! I went over CSI-473, and now I see how the `storageClassName` helps identify the specific PV.
though the `5` was only relevant for @matt-levan's test, and not to our current 1GiB PV docs.
I assume @tyichye19 did not get a `5` GiB PV if she used the documented `1` GiB value for her PV

Reply:
@oriyarde  so can I keep as-is?

Reply:
I still think the `5` should be aligned to `1`

Reply:
@oriyarde  @tyichye19  perhaps instead I should put in something more general?
"To define a specific PV size, be sure to include the `storageClassName`."

Reply:
the point here is to find a specific PV out of all the PVs that have the (already) specified size, using the relevant storageclass.
I would just leave "be sure to include the `storageClassName`"

Reply:
DONE